app/Http/Controllers/ProductController.php

```diff
@@ -3,89 +3,99 @@
 namespace App\Http\Controllers;
 
 use App\Models\Product;
+use App\Models\User;
 use Illuminate\Http\Request;
 
 class ProductController extends Controller
 {
-    public function store (Request $request)
+
+    public function store(Request $request, Product $product)
     {
+
         $request->validate([
-            'name'=>['required', 'max: 200'],
+             'name'=>['required', 'max: 200'],
              'description'=>['required', 'max: 200'],
-            'price'=>['required', 'max: 200'],
-             'qty'=>['required', 'max: 200'],
+             'price'=>['required', 'max: 200'],
         ]);
 
-        Product::create($request->all());
+
+        $product->user_id=auth()->user()->id;
+        $product->name=$request->input('name');
+        $product->description=$request->input('description');
+        $product->price=$request->input('price');
+        $product->save();
 
         return response()->json(['message'=>'Product Added Successfully!'],200);
```

Comment:
resource create response code is 201.

Read the http status codes [here](https://http.dev/status)